src/test/jtx/impl/paths.cpp

```diff
@@ -70,6 +70,13 @@ path::append_one(Account const& account)
     jv["account"] = toBase58(account.id());
 }
 
+void
+path::append_one(AccountID const& account)
+{
+    auto& jv = create();
+    jv["account"] = toBase58(account);
+}
```

Comment:
To reduce code duplication, have `append_one(Account const& account)` call this override (`append_one(account.id());`)